content/1_General/Cpp_Command.mdx

```diff
@@ -254,11 +271,11 @@ g++ -std=c++17 -O2 name.cpp -o name -Wall
 
 Explanation:
 
-- `-O2` tells `g++` to compile your code to run more quickly (see [here](https://www.rapidtables.com/code/linux/gcc/gcc-o.html))
+- `-O2` tells `g++` to compile your code to run more quickly (see [here](https://www.rapidtables.com/code/linux/gcc/gcc-o.html)). Note that `-O2` will slow down compilation time. Think of it as a trade-off between runtime and compile time. 
 - `-std=c++17` allows you to use features that were added to C++ in 2017. USACO currently uses `-std=c++11`.
 - `-Wall` checks your program for common errors. See [Debugging](/general/debugging-lang/#warning-options) for more information.
 
-You should always compile with these flags.
+You should always compile with these flags (with the possible exception of `-O2`, as described above).
```

Comment:
Is the increase in compile time so significant?

Reply:
Probably not horribly significant, revert that if you want.